.github/workflows/README.md

```diff
@@ -5,6 +5,154 @@ CI system guide
 <tr><th></th><td></td>
 -->
 
+# Introduction
```

Comment:
Thanks for this. Pleased to have some documentation for what is becoming very complex!